vck/src/commonMain/kotlin/at/asitplus/wallet/lib/cbor/CoseService.kt

```diff
@@ -186,7 +188,9 @@ class DefaultCoseService(private val cryptoService: CryptoService) : CoseService
 }
 
 class DefaultVerifierCoseService(
+    @Suppress("DEPRECATION") @Deprecated("Use verifySignature")
     private val cryptoService: VerifierCryptoService = DefaultVerifierCryptoService(),
+    private val verifySignature: VerifySignatureFun = VerifySignature.invoke(),
```

Comment:
no need for invoke if if you make invoke an `operator fun` as per my previous comment